src/researchhub_comment/serializers/rh_comment_serializer.py

```diff
@@ -164,3 +166,15 @@ def get_user_vote(self, comment):
             return vote
         except Vote.DoesNotExist:
             return None
+
+        return None
+
+    def get_review(self, comment):
+        context = self.context
+        _context_fields = context.get("rhc_dcs_get_review", {})
+        review = comment.unified_document.reviews.filter(object_id=comment.id).first()
```

Comment:
If we create a generic relation on the comment model, we can simplify this to something like
```suggestion
        review = comment.reviews.first()
```
and the model class will look something like
```
reviews = GenericRelation("review.Review")
```
You can also see an example in the bounty model class (line 60-61)